src/System.Device.Gpio/System/Device/Pwm/Drivers/Windows10PwmDriverChip.Windows.cs

```diff
@@ -22,7 +23,18 @@ public Windows10PwmDriverChip(int chipIndex)
             DeviceInformationCollection deviceInformationCollection = DeviceInformation.FindAllAsync(deviceSelector).WaitForCompletion();
             if (deviceInformationCollection.Count == 0)
             {
-                throw new ArgumentException($"No PWM device exists for PWM chip at index {chipIndex}", $"{nameof(chipIndex)}");
+                // On Hummingboard, fallback to check for generic PWM controller (friendlyname is not currently used)
+                var deviceInfo = new EasClientDeviceInformation();
+                if (deviceInfo.SystemProductName.IndexOf("Hummingboard", StringComparison.OrdinalIgnoreCase) >= 0)
+                {
+                    deviceSelector = WinPwm.PwmController.GetDeviceSelector();
+                    deviceInformationCollection = DeviceInformation.FindAllAsync(deviceSelector).WaitForCompletion();
+                }

```

Comment:
I've confirmed that the Product Name is the same inside of a container.
I don't know if there is any way to avoid this and still allow the PWM APIs to work on IoT Core images for Hummingboard